src/generic/templates/audio.js

```diff
@@ -57,12 +59,27 @@ class Audio extends Template {
     const {
       title,
       url,
+      duration,
       action,
       quickReplies,
       delay,
       fallback
     } = this
 
+    if(duration){
```

Comment:
Why do we need to add this specific check? Why can't we just include `duration` in the already present `return` statement right after this `if` condition?

Reply:
For both of the suggestion :
1. adding `duration` in return directly
2. doing `duration: duration || undefined`

is not a good design because this generates payload as:
`{ duration : undefined/null }` ,  and having that in responses where its not needed is not right. Example, this audio is generic one and no other channel has this field in the UI `duration`, so sending out a payload with this field as `undefined` doesn't make sense.

We have already seen customers complaining for such values in payload, I guess its better to avoid it.

Reply:
Cool, thanks!